src/ralph/dashboards/models.py

```diff
@@ -13,6 +14,19 @@
 )
 
 
+def _unpack_series(series):
+    if not isinstance(series, str):
+        raise ValueError('Series should be string')
+    series_field, fn = _to_pair(series, '|')
+    if (series_field != fn) and (fn != 'distinct'):
```

Comment:
we should change logic of `_to_pair` asap - this condition is terrible - couldn't `_to_pair` return `None` if there is nothing to split?